dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/programstageworkinglist/ProgramStageWorkingListControllerTest.java

```diff
@@ -261,8 +267,26 @@ void shouldDeleteWorkingListWhenDeleteRequested() {
     assertEquals(HttpStatus.OK, response.status());
   }
 
-  @Test
-  void shouldCreateWorkingListWithProgramStageQueryCriteria() {
+  static Stream<Arguments> shouldCreateWorkingListWithProgramStageQueryCriteria() {
+    Consumer<JsonProgramStageQueryCriteria> followUpIsNull =
+        json ->
+            assertFalse(
```

Comment:
I think this can be replaced with https://github.com/dhis2/dhis2-core/blob/d4c727ef6006326acb3f7d05f17be7a9d872c157/dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/JsonAssertions.java#L155